project/Dependencies.scala

```diff
@@ -1,19 +1,22 @@
-import sbt._
+import sbt.*
 
 object Dependencies {
 
-  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.4"
-  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.6"
-  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.1.4"
-  val `akka-serialization` = "com.evolutiongaming"        %% "akka-serialization" % "1.0.4"
-  val nel                  = "com.evolutiongaming"        %% "nel"                % "1.3.4"
-  val `safe-actor`         = "com.evolutiongaming"        %% "safe-actor"         % "3.0.0"
-  val `scala-tools`        = "com.evolutiongaming"        %% "scala-tools"        % "3.0.5"
-  val scalatest            = "org.scalatest"              %% "scalatest"          % "3.2.14"
+  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.5"
+  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.7"
+  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.2.0"
+  val `akka-serialization` = "com.evolutiongaming"        %% "akka-serialization" % "1.1.0"
+  val nel                  = "com.evolutiongaming"        %% "nel"                % "1.3.5"
+  val `safe-actor`         = "com.evolutiongaming"        %% "safe-actor"         % "3.1.0"
+  val `scala-tools`        = "com.evolutiongaming"        %% "scala-tools"        % "3.0.6"
+  val scalatest            = "org.scalatest"              %% "scalatest"          % "3.2.19"
   val `scala-logging`      = "com.typesafe.scala-logging" %% "scala-logging"      % "3.9.5"
+  val `scodec-bits`        = "org.scodec"                 %% "scodec-bits"        % "1.2.1"
+  val `scodec-core1`       = "org.scodec"                 %% "scodec-core"        % "1.11.10"
+  val `scodec-core2`       = "org.scodec"                 %% "scodec-core"        % "2.3.1"
 
   object Akka {
-    private val version = "2.6.19"
+    private val version = "2.6.21"
```

Comment:
I believe everyone should be on 2.6.21 already, it doesn't seem to be a problem.

Reply:
and most of our OSS base akka support libraries are on 2.6.21 for months already

Reply:
No, we use .20 via overrides. The fact that our libraries are on .21 is a problem and not a reason for others to bump it too.

Reply:
I am afraid that it will sound arrogant, but prohibiting upgrades for all other projects because your one cannot or doesn't want to update is not a solution we can accept. 

You are free to put overrides in your project or to not update some library at your own risk.

Please review changes in akka (https://github.com/akka/akka/releases/tag/v2.6.21) - they are very small and, most probably, will not have negative affect on your project.

Reply:
I stand with @mr-git here - Akka 2.6.21 is more than 1 year old now. If there is an internal reason for us to use older versions internally, there is no reason to have an outdated version in the opensource. Updating Akka 19 and 20 to 21 is trivial and by just trying to hold it in transitive dependencies is a loosing game. There are already dependencies which will bring 2.6.21 transitively (like safe-akka). If you want safeguards, they should be on the library client side - dependency overrides and such.

Reply:
@vilunov, am I right that we (including _you_) are using Akka `2.6.21` now? Shall we unblock the PR?